tests/unit/test_kubernetes_service.py

```diff
@@ -0,0 +1,325 @@
+# Copyright 2021 Canonical Ltd.
```

Comment:
I'm not sure I'd include this test file -- it's in the upstream source for the charm that hosts the library, and won't be updated automatically with `charmcraft fetch-lib`

Reply:
Done in aa98606